apache.Dockerfile

```diff
@@ -7,7 +7,8 @@
 #
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
-FROM docker.io/node:12.20.1 as builder
+
+FROM docker.io/$TARGETARCH/node:12.20.1-buster as builder
```

Comment:
Hello,
I'm not sure to understand  why we need targetarch there and buster ?

These are multi arch images already

Reply:
hello @benoitf ,

if I understand it correctly, node image `node:12.20.1` doesn't support architectures `linux/s390x` and `linux/ppc64le`. But node image `node:12.20.1-buster` supports all architectures we want to target.

Another issue here that I got following error for `linux/arm64v8` when I build image from `buster` without specifying `$TARGETARCH`:
```
...
 => ERROR [linux/arm64v8 internal] load metadata for docker.io/library/node:12.20.1-buster                                                                                                     0.7s
------
 > [linux/arm64v8 internal] load metadata for docker.io/library/node:12.20.1-buster:
------
error: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: no match for platform in manifest sha256:5407853b6aef463dc2da182751015db96958ebe5ea6ccd3bd35fd3f77f03bf8a: not found
```

Using `$TARGETARCH` resolved this issue as well.

Reply:
and using `docker.io:12.20.1-alpine3.12` ?

Reply:
let me try